include/dynamatic/Dialect/Handshake/HandshakeTypes.td

```diff
@@ -119,6 +119,103 @@ def ChannelType : Handshake_Type<"Channel", "channel", [
   }];
 }
 
+//===----------------------------------------------------------------------===//
+// Tablegen helper functions
+//===----------------------------------------------------------------------===//
```

Comment:
The way of defining functions using `class` is suggested in LLVM Documentation:
https://llvm.org/docs/TableGen/ProgRef.html#using-classes-as-subroutines